src/i18n/en-US.json

```diff
@@ -71,6 +71,10 @@
   "E2EI.updateCertificate": "Update Certificate",
   "E2EI.valid": "Valid",
   "E2EI.verified": "Verified (End-to-end Identity)",
+  "E2EI.SoftLock.Heading": "End-to-end identity certificate",
+  "E2EI.SoftLock.Content": "Your team now uses end-to-end identity to make Wire’s usage more secure. The device verification takes place automatically using a certificate.<br /><br />Enter your identity provider’s credentials in the next step to automatically get a verification certificate for this device.",
```

Comment:
There should not be html in our translations. 
We should split this into multiple translation keys